### PR TITLE
Typo fix in sonarqube/README.md

### PR DESCRIPTION
**Pull request by AdamDang**
Some sentences miss the “.” in the end.

### CODE REVIEW

File: stable/sonarqube/README.md [hunk 1]
```diff
@@ -52,6 +52,6 @@ The following table lists the configurable parameters of the Sonarqube chart and
 | `postgresql.postgresPassword`               | Postgresql database password        | `sonarPass`                                |
 | `postgresql.postgresDatabase`               | Postgresql database name            | `sonarDB`                                  |
 
-You can also configure values for the PostgreSQL database via the Postgresql [README.md](https://github.com/kubernetes/charts/blob/master/stable/postgresql/README.md)
+You can also configure values for the PostgreSQL database via the Postgresql [README.md].(https://github.com/kubernetes/charts/blob/master/stable/postgresql/README.md)
```
Comment (rjkernick) [inline stable/sonarqube/README.md]: Please remove the period, it in is the middle of a hyperlink

**Review by rjkernick — CHANGES_REQUESTED** [review]

**Comment by rjkernick** [comment]
Please resolve the merge conflict

File: stable/sonarqube/README.md (hunk 1 above)
Comment (AdamDang) [reply stable/sonarqube/README.md]: OK, It's done.

**Comment by AdamDang** [comment]
@rjkernick 
in the ci testing there is an error as following:
Checking the Chart version has increased for the chart at stable/sonarqube
Error please increment the new chart version to be greater than the existing version of 0.4.0
Exited with code 1
So I increased the version to 0.4.1

**Comment by rjkernick** [comment]
You need to rebase off of master to resolve the merge conflict. Git is stubborn sometimes.
https://help.github.com/articles/syncing-a-fork/

**Comment by AdamDang** [comment]
This PR is recreated to PR #5043.
Thanks a lot!

**Pull request closed without merging**